src/components/Book.js

```diff
@@ -13,12 +15,11 @@ const Book = ({ title, author, id }) => {
   };
 
   return (
-    <div>
-      <h2>{title}</h2>
-      <h3>{author}</h3>
-      id:
-      {id}
-      <button type="button" onClick={deleteBook} value={id}>Delete</button>
+    <div className="book font2">
+      <text className="category font1">{category}</text>
+      <h2 className="title">{title}</h2>
+      <h3 className="author">{author}</h3>
+      <button className="delete" type="button" onClick={deleteBook} value={id}>Remove</button>
     </div>
   );
```

Comment:
[REQUIRED] I that the previous reviewer didn't go into details. But what he means by looks a lot different includes the blue colour of the author's name, the percentage circle and every other thing in the Zeplin design irrespective of whether you created the functionality for them or not

| Zeplin Template|
|---------------|
|![image](https://user-images.githubusercontent.com/62938735/192120355-b2008510-f4f5-4cb0-b91d-9bb7ec92b505.png)|